package.json

```diff
@@ -58,6 +57,19 @@
     "typescript": "^5.4.5",
     "typescript-eslint": "^7.9.0"
   },
+  "release": {
+    "branches": [
+      "main",
+      {
+        "name": "develop",
+        "channel": "alpha",
+        "prerelease": "alpha"
+      }
+    ]
+  },
+  "multi-release": {
+    "sequentialInit": true
```

Comment:
Why we dont need this?
```
    "deps": {
      "bump": "override",
      "release": "patch"
    }
 ```

Reply:
@DiRaiks 
These are default values:
- https://github.com/dhoulb/multi-semantic-release?tab=readme-ov-file#cli
- https://github.com/dhoulb/multi-semantic-release/blob/e649d21287cd5a88bc642302af09e34830c15e83/lib/updateDeps.js#L145
So, I guess, no need to set them explicitly